sunpy/visualization/animator/wcs.py

```diff
@@ -281,6 +282,8 @@ def plot_start_image_2d(self, ax):
                        'origin': 'lower'}
         imshow_args.update(self.imshow_kwargs)
 
+        imshow_args['vmin'], imshow_args['vmax'] = self.get_2d_plot_limts()
```

Comment:
That reminds me, what if someone added "vmax" or "vmin", would they not be replaced with None by this? 
Maybe we do need the check?

Reply:
Yes, I was just about to being with the tests and I thought of this as well :sweat_smile:, the check first check for 'vmax' in ```self.imshow_kwargs``` prevents this from happening